components/web/apps/org.wso2.carbon.apimgt.admin.app/src/main/fragments/policy-add/public/css/custom.css

```diff
@@ -0,0 +1,324 @@
+.api-wrapper {
+	background-color: #EFEFEF;
+	position: relative;
+}
+.center-wrapper{
+    margin:0 auto;
+    margin-left:auto;
+    margin-right:auto;
+    align:center;
+    text-align:center;
+    padding-top: 100px;
+    padding-bottom: 100px;
+}
+
+input.ep-field{
+    min-width: 50%;
+}
+._jsonform-array-ul label{
+    width:118px;
+    margin-left: 0px;
+}
+._jsonform-array-ul label .help_popup{
+    display: none;
+}
+#description {
+    resize: vertical;
+}
+.modal-content{
+    background: #FFF;
+    color:#000;
+ }
+ .modal-header h3{
+    padding: 0px;
+ }
+ .modal-header{
+    background: #f1f1f1;
+    margin-bottom: 20px;
+    padding: 15px;
+    border-bottom: 1px solid #ddd;
+ }
+ .modal-header .close{
+    color:#222;
+    margin-top: 6px;
+    font-size: 16px;
+ }
+#pipeline-content .panel-group{
+    margin-bottom:0px;
+}
+.navbar-menu-toggle,.navbar-menu-toggle:hover, .navbar-menu-toggle:focus, .navbar-menu-toggle:active{
+    background:#222;
+}
+.navbar-menu-toggle[aria-expanded=true]{
+    background:#242932;
+}
+.form-group label>.requiredAstrix {
```

Comment:
Shall we follow the same convention here instead of camel case. 
required-astrix instead of requiredAstrix